pkg/_pub_shared/lib/format/encoding.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:typed_data';
+
+/// Base64 encodes [dataPoints] as 32-bit unsigned integers serialized with
+/// big endian byte order.
+String encodeIntsAsBigEndianBase64String(List<int> dataPoints) {
```

Comment:
nit: rename here and the other `...BigEndian` -> `...LittleEndian`.

Reply:
done